tg_fm_news.py

```diff
@@ -26,8 +36,22 @@ def print_current_news(interval):
             item_text = item.text
             item_href = "https://sputnik.by/geoBelarus" + item.get("href")
             all_news_dict[item_text] = item_href  # Запись значений в словарь
-            bot.send_message(chat_id, f"{item_text}: {item_href}")
-        # bot.send_message(chat_id, f"{item_text}: {item_href}")
+            # print(f"{item_text}: {item_href}")
+            data = item_text
+            cursor.execute("SELECT * FROM news WHERE heading = %s", (data,))
+            res = cursor.fetchone()
+
+            if res:
+                print(f"В наличии в БД. Время обновления: {datetime.datetime.now()} ")
+            else:
+                print(f"НЕ в наличии,сейчас добавим и отправим в ТГ-бот.  {datetime.datetime.now()} ")
+                news = (item_text,
+                        item_href,
+                        datetime.datetime.now())
+                cursor.execute("INSERT INTO news (heading, url, date) VALUES (%s, %s, %s)", news)
```

Comment:
Save to the database AFTER sending to tg.

Reply:
Fixed